tests/integration/cmd_pref_config_test.go

```diff
@@ -234,25 +234,36 @@ var _ = Describe("odo preference and config command tests", func() {
 	})
 
 	Context("when viewing local config without logging into the OpenShift cluster", func() {
-		It("should list config successfully", func() {
-			helper.Cmd("odo", "create", "--s2i", "nodejs", "--git", "https://github.com/openshift/nodejs-ex", "--project", commonVar.Project, "--context", commonVar.Context).ShouldPass()
-			helper.Cmd("odo", "config", "set", "--env", "hello=world", "--context", commonVar.Context).ShouldPass()
-			kubeconfigOld := os.Getenv("KUBECONFIG")
-			os.Setenv("KUBECONFIG", "/no/such/path")
-			configValue := helper.Cmd("odo", "config", "view", "--context", commonVar.Context).ShouldPass().Out()
-			helper.MatchAllInOutput(configValue, []string{"hello", "world"})
-			os.Setenv("KUBECONFIG", kubeconfigOld)
+		var ocRunner helper.OcRunner
+		var token string
+		BeforeEach(func() {
+			helper.CopyExample(filepath.Join("source", "nodejs"), commonVar.Context)
+			helper.Cmd("odo", "create", "--s2i", "nodejs", "nodejs", "--project", commonVar.Project, "--context", commonVar.Context).ShouldPass()
+			ocRunner = helper.NewOcRunner("oc")
+			token = ocRunner.GetToken()
+			ocRunner.Logout()
```

Comment:
Does it have to be `oc`? Can we not use a generic CLI runner?

Reply:
Until I can figure out how to simulate logout with `kubectl`, this is the only way